locale/de_DE/LC_MESSAGES/django.po

```diff
@@ -17,62 +17,70 @@ msgstr ""
 "X-Crowdin-File: /[bookwyrm-social.bookwyrm] main/locale/en_US/LC_MESSAGES/django.po\n"
 "X-Crowdin-File-ID: 1553\n"
 
-#: bookwyrm/forms.py:365
+#: bookwyrm/forms.py:239
+msgid "Domain is blocked. Don't try this url again."
+msgstr "Die Domäne ist blockiert. Versuchen Sie diese Url nicht mehr."
```

Comment:
Thank you so much for translating these! Would you be down to add them at https://translate.joinbookwyrm.com/ instead? That's the canonical source of translation strings, and I think it will overwrite changes you make locally like this. 

(I'd also be happy to copy it over if you would prefer that)

Reply:
I will gladly participate in the translation, but until I get to it, I would ask you to copy this translation in there.